app/src/main/java/com/vci/vectorcamapp/core/data/room/migrations/versions/Migration_5_6_AddUploadStatusToSpecimen.kt

```diff
@@ -0,0 +1,12 @@
+package com.vci.vectorcamapp.core.data.room.migrations.versions
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+
+val MIGRATION_5_6_ADD_UPLOAD_STATUS_TO_SPECIMEN = object : Migration(5, 6) {
+    override fun migrate(db: SupportSQLiteDatabase) {
+        db.execSQL(
+            "ALTER TABLE specimen ADD COLUMN uploadStatus TEXT NOT NULL DEFAULT 'NOT_STARTED'"
+        )
```

Comment:
The formatting of this string is not the same as in the other files. Column names and table names should be wrapped in ``. This might cause 6.json to change (lets chat if that happens).